src/corecel/sys/Device.cc

```diff
@@ -317,6 +320,23 @@ void activate_device(Device&& device)
     CELER_DEVICE_CALL_PREFIX(SetDevice(device.device_id()));
     d = std::move(device);
 
+    // Check capability version against cmake variable; rough but better than
+    // nothing! CMake format: "70-real 72-virtual" or "35;50;72" or "
```

Comment:
... "or gfx90a"

Reply:
and currently we would issue a spurious message for the `gfx` case, right?

Reply:
No, we search for "90" in "gfx90a" so it works. Same with other mixed architectures